src/views/RecIM/components/List/index.jsx

```diff
@@ -26,7 +26,7 @@ const ParticipantList = ({
       <ParticipantListing
         key={participant.username}
         participant={participant}
-        minimal={minimal}
+        minimal={minimal || participant.Role === 'Team-captain/Creator'} //temporarily disable captains from leaving or changing their status
```

Comment:
Interesting, good change! The naming on that variable should change though, and I think changing the comment to just remain there (so remove the temporarily) would be good. While that might be a little out of scope of the PR, it is something we have been meaning to do so it would be great if it could be now.

Reply:
I've made it an issue to be handled better. But for now, without those usecases, we shouldn't allow a team to accidentally have no people.

Reply:
Ok, sounds good!